tests/sanity/tests/collaborative/issues.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test } from '@playwright/test'
+import { generateId, PlatformURI, PlatformUser, PlatformUserSecond } from '../utils'
+import { SelectWorkspacePage } from '../model/select-workspace-page'
+import { allure } from 'allure-playwright'
+import { LoginPage } from '../model/login-page'
+import { NewIssue } from '../model/tracker/types'
+import { IssuesPage } from '../model/tracker/issues-page'
+import { LeftSideMenuPage } from '../model/left-side-menu-page'
+import { IssuesDetailsPage } from '../model/tracker/issues-details-page'
+
+test.describe('Collaborative test for issue', () => {
+  test.beforeEach(async ({ page }) => {
+    await allure.parentSuite('Collaborative test')
+    await (await page.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+  })
+
+  test('Issues can be assigned to another users', async ({ browser }) => {
+    const userFirstContext = await browser.newContext()
```

Comment:
Why not use 
```typescript
const page2 = await browser.newPage({
      storageState: PlatformSetting2
    })
```
In this case it will not be required to do login each time.

Reply:
checking

Reply:
done